src/catalog/src/remote/manager.rs

```diff
@@ -192,6 +208,24 @@ impl RemoteCatalogManager {
         info!("Created catalog '{catalog_key}");
         Ok(catalog_provider)
     }
+
+    async fn invalidate_cache(&self, catalog: &str, schema: &str, table: &str) -> Result<()> {
+        let schema_provider = self
+            .schema(catalog, schema)
+            .await?
+            .context(SchemaNotFoundSnafu { catalog, schema })?;
```

Comment:
In the case of cache expiration, this will cause a remote call to metasrv.
We should need a CacheManager, specifically responsible for invalidating cache.